content/en/blog/2025/opentelemetry-logging-and-you.md

```diff
@@ -0,0 +1,93 @@
+---
+title: OpenTelemetry Logging and You
+linkTitle: OTel Logging
+date: 2025-04-16
+author: >-
+  [Austin Parker](https://github.com/austinlparker) (honeycomb.io)
+sig: Logs
+---
+
+If you’ve been following OpenTelemetry for a while, you’ve probably heard a lot
+about logs. [Log bridges](/docs/specs/otel/glossary/#log-appender--bridge),
+[Logs API](/docs/specs/otel/logs/api), [Events](/docs/concepts/glossary/#event),
+you name it, we’ve talked about it. This blog post is intended to be a
+discussion of the rationale and current design direction of logging in
+OpenTelemetry.
+
+## Definitions
+
+Let’s get started with a basic definition of how OpenTelemetry thinks about
+logs. Broadly, logs are any telemetry that is emitted through a LoggerProvider,
+and are created by calling the Logs API. There are two ways we intend for users
+to do this –
+
+- Use the Logs API as a sink for existing loggers (sending existing logs to
+  OpenTelemetry)
+- Use the Logs API to emit [Events](/docs/concepts/glossary/#event) or log
+  records.
+
+Regardless of the source, all logs emitted through the Logs API are able to
+participate in the OpenTelemetry Context. Log records share the same transaction
+or thread context as a metric or span. This is a fundamental part of
+OpenTelemetry's design -- all of your telemetry signals can be correlated
+through context.
+
+With that said, what’s the distinction between a log and an event? It is rather
+straightforward – events are logs that
+[OpenTelemetry can make guarantees about](/docs/specs/otel/logs/data-model/#events).
+An Event has a mandatory name and a structure that is defined via a schema,
+similar to how semantic conventions are defined today. While all log records in
+OpenTelemetry are structured, only Events have this defined schema. We are using
+the word event because it most clearly describes what an event is – something
+that happens, without a duration, and can be named.
+
+**We believe that most log records should be events.**
+
+## How is this different from other signals?
+
+Logs and Spans are often thought of as being conceptual twins, especially when
+you start talking about events. What is a span other than an event with a
+particularly detailed schema, after all? A key distinction between them is that
+**spans have durations**. Events make no guarantees or claims about duration or
+time; They could represent a truly instantaneous occurrence, or the result of
+minutes or hours of computation. The other big difference is that **spans have
+an explicit hierarchy**. A span is part of a trace that contains many spans. A
+span with no children is, in and of itself, a complete trace. Events do not have
+this guarantee.
```

Comment:
this was a bit confusing to me

> A span with no children is, in and of itself, a complete trace.

A span with no children and no parent?

> Events do not have this guarantee.

it wasn't clear what guarantee?

does it help to just say that spans can have children?